static/js/portico/signup.js

```diff
@@ -196,4 +196,9 @@ $(function () {
     $("body").on("click", "#choose_email .choose-email-box", function () {
         this.parentNode.submit();
     });
+
+    $('#id_terms').on('change', function () {
+        document.getElementById('register-button').disabled = !this.checked;
```

Comment:
I think you should store the result of the first query in a variable instead of doing two queries here.